src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -16,15 +16,14 @@ namespace Stratis.Bitcoin.Features.BlockStore
     /// <summary>
     /// <see cref="IBlockRepository"/> is the interface to all the logics interacting with the blocks stored in the database.
     /// </summary>
-    /// <seealso cref="System.IDisposable" />
-    public interface IBlockRepository : IBlockStore, IDisposable
+    public interface IBlockRepository : IBlockStore
     {
         /// <summary>
         /// Persist the next block hash and insert new blocks into the database.
```

Comment:
It looks like you have changed the argument but not the summary

Reply:
what do you mean?

Reply:
You need to update a comment Ofthe summary